content/blog/introducing-crd2pulumi/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Introducing crd2pulumi: Typed CustomResources for Kubernetes"
 date: 2020-08-12
-meta_desc: Generate Kubernetes CustomResource types in TypeScript, Python, C#, and Go.
+meta_desc: Generate typed Kubernetes CustomResource in TypeScript, Python, C#, and Go.
```

Comment:
This doesn't read smoothly to me. Was there a problem with the existing description?

Reply:
I felt that "typed Kubernetes CustomResources" was a bit more technically accurate, since the the CustomResource itself is a class and not a type in TypeScript/C#/Python, but I'll change it back to "CustomResource types" since I agree it reads more smoothly